.packit.yaml

```diff
@@ -10,6 +10,7 @@ upstream_project_name: packitos
 downstream_package_name: packit
 current_version_command: ["python3", "setup.py", "--version"]
 create_tarball_command: ["python3", "setup.py", "sdist", "--dist-dir", "."]
+create_pr: false
```

Comment:
I actually debate whether this is the route we want to pick, but given the fact that we are still unable to create dist-git PRs to this day, the proposed solution seems like an easy way to globally configure PR/no-PR, so I'm +1 on this.